src/core/iomux/epoll_wait_call.cpp

```diff
@@ -405,13 +404,12 @@ int epoll_wait_call::ring_poll_and_process_element()
     return m_epfd_info->ring_poll_and_process_element(&m_poll_sn_rx, &m_poll_sn_tx, nullptr);
```

Comment:
[question] do we still need "poll sn" logic?

Reply:
Removed all leftovers of poll sn